src/components/Reviews.jsx

```diff
@@ -1,19 +1,25 @@
 import { useEffect, useState } from "react";
 import { getReviews } from "../tools/api";
 import ReviewUnit from "./ReviewUnit";
-import { Link } from "react-router-dom";
+import { Link, useParams } from "react-router-dom";
 import Loading from "./Loading";
 
 export default function Reviews() {
   const [reviews, setReviews] = useState([]);
   const [loading, setLoading] = useState(true);
+  const { category } = useParams();
 
   useEffect(() => {
     getReviews().then((res) => {
-      setReviews(res);
+      if (category) {
+        let revByCat = res.filter((review) => review.category === category);
+        setReviews(revByCat);
```

Comment:
The filtering of reviews should happen in the request from back end with queries, not on the front end.